components/combobox/stories/combobox.stories.ts

```diff
@@ -0,0 +1,867 @@
+    <auro-menuoption value="Prefer Alaska" id="option-5">Prefer Alaska</auro-menuoption>
+    <auro-menuoption static nomatch>No matching option</auro-menuoption>
+  </auro-menu>
+</auro-combobox>
+  `,
+};
+
+// TODO: This is not functioning
+// TODO: How to translate this to template expressions
+//       (https://lit.dev/docs/templates/expressions/#event-listener-expressions),
+//       as in checkboxStories.ResetState?
+export const Loading: Story = {
+  render: () => html`
+<auro-combobox id="loadingExample">
+  <span slot="bib.fullscreen.headline">Loading Combobox Header</span>
+  <span slot="label">Please select a preference</span>
+  <auro-menu id="loadingExampleComboboxMenu">
+    <auro-loader slot="loadingIcon" orbit xs></auro-loader><span slot="loadingText">Loading...</span>
+  </auro-menu>
+</auro-combobox>
+
+<script>
+(function() {
+  const combobox = document.querySelector("#loadingExample");
+  const menu = document.querySelector("#loadingExampleComboboxMenu");
+
+  function emptyMenu() {
+    const menuoptions = menu.querySelectorAll('auro-menuoption');
+    menuoptions.forEach(mo => menu.removeChild(mo));
+  }
+
+  function fillMenu() {
+    menu.innerHTML += \`
+      <auro-menuoption value="Apples" id="option-0">Apples</auro-menuoption>
+      <auro-menuoption value="Oranges" id="option-1">Oranges</auro-menuoption>
+      <auro-menuoption value="Peaches" id="option-2">Peaches</auro-menuoption>
+      <auro-menuoption value="Grapes" id="option-3">Grapes</auro-menuoption>
+      <auro-menuoption value="Cherries" id="option-4">Cherries</auro-menuoption>
+      <auro-menuoption static nomatch>No matching option</auro-menuoption>
+    \`;
+  }
+
+  function load(menu) {
+    clearTimeout(load.id);
+    emptyMenu();
+    menu.setAttribute('loading', 'loading');
+    load.id = setTimeout(() => {
+      menu.removeAttribute('loading');
+      fillMenu();
+    }, 1000);
+  }
+
+  combobox.addEventListener("input", (e) => {
+    if (e.target.value && e.target.value !== e.target.optionSelected?.textContent) {
+      load();
+    }
+  });
+})();
+</script>
+  `,
+};
+
+// TODO: Apply useful viewport dimensions
+export const FullscreenBreakpoint: Story = {
+  render: () => html`
+<auro-combobox fullscreenBreakpoint="lg">
+  <span slot="bib.fullscreen.headline">Bib Header</span>
+  <span slot="label">Name</span>
+  <auro-menu>
+    <auro-menuoption value="Apples" id="option-0">Apples</auro-menuoption>
+    <auro-menuoption value="Oranges" id="option-1">Oranges</auro-menuoption>
+    <auro-menuoption value="Peaches" id="option-2">Peaches</auro-menuoption>
+    <auro-menuoption value="Grapes" id="option-3">Grapes</auro-menuoption>
+    <auro-menuoption value="Cherries" id="option-4">Cherries</auro-menuoption>
+    <auro-menuoption static nomatch>No matching option</auro-menuoption>
+  </auro-menu>
+</auro-combobox>
+  `,
+};
+
+// TODO: How to register auro-dialog?
```

Comment:
Some stories, like this one, use the `<auro-dialog>` component, but it was not available in `@aurodesignsystem` and I could not find instructions for using it.

As such, they do not currently function.